wpeverest/user-registration
# Fix- Email sender option value while sending a test email

### All Submissions:






### Changes proposed in this Pull Request:



Previously, while sending a test email the Email sender option section value like From Address and Email are not showing instead WordPress core address and email is showing in reply. This PR will fix this issue.
![image](https://user-images.githubusercontent.com/34309829/153343138-6e007e35-a42e-4aa2-bcae-95003e0789d4.png)


### How to test the changes in this Pull Request:

1. Navigate to the User Registration settings and then to the Emails tab.
2. You can see the Email sender options and provide the desired "From" name and Address.
3. then In the Send Test Email to which address you want to send a test email. 
4. verify that the test email is from the saved email sender options in your received email address.

### Types of changes:




### Other information:




### Changelog entry

> Fix- Email sender option value while sending a test email.
